### PR TITLE
[STARTAI-50] Added visualisation for our time series

### PR DESCRIPTION
**Pull request by ArsenOOO7**

### CODE REVIEW

**Review by nasstis — APPROVED** [review]

File: vizual/vizual_lib.py [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+
+        ordinal_encoder = OrdinalEncoder()
+
+        transformed_X = self.__data.copy()
+        transformed_X[object_columns] = ordinal_encoder.fit_transform(transformed_X[object_columns])
+
+        self.__data = transformed_X
+
+    # @staticmethod
+    # def vizual_of(old, data, target_column: str = None, transform: bool = None):
```
Comment (JJerome-NM) [inline vizual/vizual_lib.py]: Remove this plz

**Review by JJerome-NM — CHANGES_REQUESTED** [review]

File: vizual/vizual_lib.py (hunk 1 above)
Comment (ArsenOOO7) [reply vizual/vizual_lib.py]: no

**Review by JJerome-NM — APPROVED** [review]
OK!!!!!!!!!!!!!!!

File: vizual/vizual_lib.py (hunk 1 above)
Comment (ArsenOOO7) [reply vizual/vizual_lib.py]: yes

**Pull request merged**